components/suggestion/suggestion.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Modifier} from 'react-day-picker';
+
+import {UserProfile} from './command_provider/app_command_parser/app_command_parser_dependencies';
+
+type Props = {
+    onClick?: (term: string, matchedPretext: string) => boolean;
+    onMouseMove?: (term: string) => void;
+    term: string;
+    matchedPretext: string;
+    isSelection?: boolean;
+    item: any;
```

Comment:
what should be a better type here.
Item is being used in many provider files.
Its type is different in almost each file.